templates/signed-out.html

```diff
@@ -1,11 +1,33 @@
-{% extends 'layouts/_base.html' %}
+{% extends 'errors/_base.html' %}
 
-{% set page_title = _("Signed out") %}
+{% set page_title = _("Save and sign out confirmation") %}
```

Comment:
Was this requested?. I believe they were happy with `Signed out` when I checked.

Reply:
Cant remember why I changed this. Think it just made more sense to me. Ive reverted it